src/main/java/org/usfirst/frc4904/robot/commands/RaiseHook.java

```diff
@@ -0,0 +1,34 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+
+public class RaiseHook extends MotorConstant {
+  public static final double DEFAULT_UP_SPEED = 0; // TODO: This is an untested value
+  protected Motor tubeMotor;
+  protected double speed;
+
+  /**
+   * Raise the deploy tube to hook onto the generator switch
+   * 
+   * @param tubeMotor The motor to use to raise
+   * @param speed     The speed to raise the deploy tube at
+   */
+  RaiseHook(Motor tubeMotor, double speed) {
+    super("RaiseHook", tubeMotor, speed);
+  }
+
+  /**
+   * Raise the deploy tube to hook onto the generator switch
+   * 
+   * @param tubeMotor The motor to use to raise
+   */
+  RaiseHook(Motor tubeMotor) {
+    this(tubeMotor, DEFAULT_UP_SPEED);
+  }
+
+  @Override
+  public void execute() {
+    tubeMotor.set(speed);
+  }
```

Comment:
same as in the last file

Reply:
the `execute` method can be deleted because its just rewriting existing functionality